jenkins-builds/7.0.UP02-21.02-dom-gpu

```diff
@@ -41,17 +43,21 @@
  NCL-6.4.0.eb                                           --set-default-module
  NCO-4.8.1-CrayGNU-21.02.eb                             --set-default-module
  ncview-2.1.7-CrayGNU-21.02.eb                          --set-default-module
+ netcdf-python-1.5.4-CrayGNU-21.02-python3.eb           --set-default-module
  numpy-1.17.2-CrayGNU-21.02.eb                          --set-default-module
+ OOOPS-1.0.eb
```

Comment:
I could not find doc on the user portal hence I would rather remove it before we show this to the world.

Reply:
I fear that the world already knows about it:
```
daint101:~$ module load daint-gpu
daint101:~$ module avail OOOPS

--------------------------------- /apps/daint/UES/jenkins/7.0.UP02-20.11/gpu/easybuild/modules/all ----------------------------------
OOOPS/1.0
```
I think that @victorusu is working on the documentation.